app/src/main/java/com/example/plantsapp/presentation/ui/loading/LoadingDialog.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.plantsapp.presentation.ui.loading
+
+import android.app.AlertDialog
+import android.content.Context
+import com.example.plantsapp.R
+
+class LoadingDialog(context: Context) {
+
+    private val alertDialog by lazy {
+        AlertDialog.Builder(context)
+            .setView(R.layout.dialog_loading)
```

Comment:
- Probably you do not need here lazy
- Also shat will happen if I click outside of alert while loading is still in progress?

Reply:
> * Probably you do not need here lazy
> * Also shat will happen if I click outside of alert while loading is still in progress?

2 - the dialog dismisses..

Reply:
> > * Probably you do not need here lazy
> > * Also shat will happen if I click outside of alert while loading is still in progress?
> 
> 2 - the dialog dismisses..

fixed this